pglogrepl.go

```diff
@@ -283,6 +283,230 @@ func StartReplication(ctx context.Context, conn *pgconn.PgConn, slotName string,
 	}
 }
 
+type BaseBackupOptions struct {
+	// Request information required to generate a progress report, but might as such have a negative impact on the performance.
+	Progress          bool
+	// Sets the label of the backup. If none is specified, a backup label of 'wal-g' will be used.
+	Label             string
+	// Request a fast checkpoint.
+	Fast              bool
+	// Include the necessary WAL segments in the backup. This will include all the files between start and stop backup in the pg_wal directory of the base directory tar file.
+	Wal               bool
```

Comment:
I think that this should be `WAL` to conform to Go naming conventions for acronyms.

Reply:
Agree. CHanged.